src/features/list/dash.ts

```diff
@@ -22,7 +23,10 @@ import { IFeaturesObject } from "../types";
  * @param {Object} features
  */
 function addDASHFeature(features : IFeaturesObject) : void {
-  features.transports.dash = dash;
+  if (!features.transports.hasOwnProperty("dash")) {
```

Comment:
Isn't it sufficient to check if the property is not defined here ?

Reply:
Yeah maybe, don't remember why I did that